apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -954,6 +957,13 @@ type ResourcePlacementStatus struct {
 	// +kubebuilder:validation:MaxItems=100
 	DiffedPlacements []DiffedResourcePlacement `json:"diffedPlacements,omitempty"`
 
+	// ObservedResourceIndex is the index of the resource snapshot that is currently being rolled out to the given cluster.
+	// During rollout, depending on the rollout strategy, clusters may observe different resource indices.
+	// ObservedResourceIndex is the resource snapshot index observed by the conditions in the ResourcePlacementStatus.
```

Comment:
not clear what this means

Reply:
I was following the wording on line 826. It means the conditions of the member cluster are based on this `ObservedResourceIndex`, whether this version of data is rolled out, applied, and available.

Reply:
For updateRun, it will be whichever version that currently exists on the member cluster.

Reply:
Updated, hope it's clearer now.